package.json

```diff
@@ -18,6 +18,7 @@
     "email-validator": "^2.0.4",
     "express": "^4.18.2",
     "generate-password": "^1.7.1",
+    "libphonenumber-js": "^1.11.4",
```

Comment:
I want to change a library for phone validating, so I install new one

Reply:
Is it the final library already, or you just plan to add one?

Reply:
Yes, it is final library. The `phone` library will be deleted

Reply:
Ok, np